.gitignore

```diff
@@ -2,6 +2,7 @@
 coverage
 spec/examples.txt
 demo/.env
+ca_bundles/*
```

Comment:
hmm can we put this in a `var/` subdirectory? May also be a consideration for @byucesoy's unix sockets.

Reply:
Makes sense. I'll open a PR to update sockets' location.